fe/fe-core/src/main/java/com/starrocks/common/proc/BackendsProcDir.java

```diff
@@ -81,7 +82,8 @@ public class BackendsProcDir implements ProcDirInterface {
         builder = new ImmutableList.Builder<String>()
                 .addAll(TITLE_NAMES)
                 .add("StarletPort")
-                .add("WorkerId");
+                .add("WorkerId")
+                .add("WarehouseName");
```

Comment:
‌‌If in share-nothing mode, what is the value of WarehouseName column?

Reply:
default_warehouse